horreum-web/src/domain/schemas/FindUsagesModal.tsx

```diff
@@ -61,7 +61,7 @@ function usageToCells(u: LabelLocation) {
                     Report config <b>{loc.title}</b> in {loc.where} {loc.name ? loc.name : ""}
                 </DataListCell>,
                 <DataListCell key={1}>
-                    <ButtonLink to={`/reports/table/config/${loc.configId}`}>Go to</ButtonLink>
+                    <ButtonLink to={`/test/${loc.testId}/reports/table/config/${loc.configId}`}>Go to</ButtonLink>
```

Comment:
This is fixing just one issue, the other one (`loc.configId` undefined) would require much effort and I think it never worked. Created separate issue for that https://github.com/Hyperfoil/Horreum/issues/1682.